Examples/UITestBed/UITestBed-UITests/UITestBed.strings

```diff
@@ -0,0 +1,38 @@
+/* 
+  UITest.strings
+  UITestBed
+
+  Created by Edward on 5/7/18.
+  Copyright © 2018 Branch. All rights reserved.
+*/
+
+"UITestSafari" =
+"
+{
+  \"~referring_link\" : \"https:\\/\\/branch-uitestbed.app.link\\/YZIfk1NgJM\",
+  \"~creation_source\" : 3,
+  \"$one_time_use\" : false,
+  \"+is_first_session\" : false,
```

Comment:
Can this be always `false` in the test environment?

Reply:
If a special test was constructed, then yes. But in this case it will be no.